mods/seasonal/moves.js

```diff
@@ -135,6 +135,48 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Bug",
 	},
+	// Brandon
+	blusterywinds: {
+		accuracy: 100,
+		basePower: 60,
+		category: "Special",
+		desc: "",
+		shortDesc: "",
+		id: "blusterywinds",
+		name: "Blustery Winds",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			// delta stream unable to be done without adding the weather for the move duration then removing it after
+			this.add('-anim', source, "Defog", target);
+		},
+		onHit: function (target, source, move) {
+			/**@type {?boolean | number} */
```

Comment:
Isn't used as a number though

Reply:
weirdly enough that's how defog's success variable is type scripted, i c/p'd the move to make it easier in this case. I changed it on my editor at the time of this comment and ran npm test, seems removing the `` | number`` creates no issue. I can remove it if you want.